src/main/java/codesquad/fineants/domain/fcm_token/FcmToken.java

```diff
@@ -0,0 +1,39 @@
+package codesquad.fineants.domain.fcm_token;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import codesquad.fineants.domain.BaseEntity;
+import codesquad.fineants.domain.member.Member;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Entity
+public class FcmToken extends BaseEntity {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "member_id")
+	private Member member;
+
+	private String token;
+
```

Comment:
```suggestion
private LocalDateTime latestActivationTime
```
해당 컬럼은 FCM Token을 가지고 최근에 활성화된 시간을 의미합니다. 해당 데이터를 이용하여 오래된 FCM 토큰을 제거할 용도로 사용됩니다.